### PR TITLE
compile releases with statically with openssl

### PR DESCRIPTION
**Pull request by karthik2804**

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -42,20 +42,24 @@ jobs:
...
           - {
-              os: "ubuntu-latest",
+              os: "ubuntu-20.04",
               arch: "amd64",
               wasiSDK: "linux",
               extension: "",
-              buildArgs: "",
+              # Ubuntu 22.04 no longer ships libssl1.1, so we statically
+              # link it here to preserve release binary compatibility.
+              buildArgs: "\"--features openssl/vendored\"",
```
Comment (lann) [inline .github/workflows/release.yaml:51]: Is `spin-js-sdk` actually using openssl? :thinking:

Comment (lann) [reply .github/workflows/release.yaml:51]: It doesn't appear in the `Cargo.lock`...

Comment (karthik2804) [reply .github/workflows/release.yaml:51]: I just looked as well 😅

Comment (karthik2804) [reply .github/workflows/release.yaml:51]: I am going to close this PR.

**Pull request closed without merging**